proposals/lip-0049.md

```diff
@@ -401,26 +441,17 @@ registrationCCMParamsSchema = {
     "properties": {
         "networkID": {
```

Comment:
Not necessary any more as `chainID == networkID`.

Reply:
already removed